presto-spark-classloader-interface/src/main/java/com/facebook/presto/spark/classloader_interface/SerializedTaskInfo.java

```diff
@@ -20,15 +20,25 @@
 public class SerializedTaskInfo
         implements Serializable
 {
+    private final int fragmentId;
+    private final int taskId;
     private final byte[] bytes;
 
-    public SerializedTaskInfo(byte[] bytes)
+    public SerializedTaskInfo(int fragmentId, int taskId, byte[] bytes)
     {
+        this.fragmentId = fragmentId;
+        this.taskId = taskId;
         this.bytes = requireNonNull(bytes, "bytes is null");
     }
 
     public byte[] getBytes()
     {
         return bytes;
     }
+
+    @Override
+    public String toString()
+    {
+        return fragmentId + "." + taskId;
```

Comment:
is this for debug?

Reply:
This is to avoid `byte[]` being displayed on the Spark UI